src/test/integration/README.md

```diff
@@ -20,4 +20,25 @@ Successfully passing all the tests with the command above would output this resu
 
 ![success_test](images/successful-test.png)
 
-Python Version: 2.7.10
\ No newline at end of file
+Python Version: 2.7.10
+
+### Docker
+
+This directory contains files that run integration tests against the Locations Frontend API.
+
+First, create a configuration.json file from configuration_example.json.
+
+Next, use these commands to build and run the container. All you need installed is Docker.
+
+```shell
+$ docker build -t location-frontend-tests .
+# Run the integration tests in Unix
+$ docker run -v "$PWD"/configuration.json:/usr/src/app/configuration.json:ro location-frontend-tests
+# Run the integration tests in Windows
+$ docker run -v c:\path\to\configuration.json:/c:\usr\src\app\configuration.json:ro location-frontend-tests
+```
+
+Currently, there is one failure test which will return wrong error code for POST method:
```

Comment:
Since we are expecting that test to fail, we should skip it until the root of the issue is fixed. Could you skip the test_verbs test and display a message like this?
'Apigee cache bug - JIRA ticket CO-699'